openstack/compute/v2/extensions/rescueunrescue/results.go

```diff
@@ -0,0 +1,18 @@
+package rescueunrescue
+
+import "github.com/gophercloud/gophercloud"
+
+// RescueResult is the response from a Rescue operation. Call its ExtractErr
+// method to determine if the call succeeded or failed.
+type RescueResult struct {
+	gophercloud.ErrResult
```

Comment:
I realize this is how `RescueResult` is currently implemented, but I wonder if we can take the opportunity to make this more aligned with the rest of Gophercloud.

There are probably a few edge cases, but normally `ErrResult` is for responses that wouldn't contain a body - only an error.

What are your thoughts on making this `gophercloud.Result`?

Reply:
Yes, it seems more appropriate, I'll change it.